src/test/java/io/vertx/uritemplate/ExpansionTest.java

```diff
@@ -77,7 +77,7 @@ public void testSimpleStringExpansion() {
     assertEquals("one=1,two=2,three=3,comma=%2C", UriTemplate.of("{map*}").expandToString(variables));
     assertEquals("", UriTemplate.of("{empty_map}").expandToString(variables));
     assertEquals("", UriTemplate.of("{empty_map*}").expandToString(variables));
-    assertExpansionFailure("{list:1}");
```

Comment:
I belive this is a bug in the current implementation and the updated result is the expected one.